tests/jenkins.test.ps1

```diff
@@ -0,0 +1,163 @@
+            @{Name = 'checks-api' }
+            @{Name = 'cloudbees-folder' }
+            @{Name = 'command-launcher' }
+            @{Name = 'credentials' }
+            @{Name = 'credentials-binding' }
+            @{Name = 'display-url-api' }
+            @{Name = 'durable-task' }
+            @{Name = 'echarts-api' }
+            @{Name = 'email-ext' }
+            @{Name = 'font-awesome-api' }
+            @{Name = 'git' }
+            @{Name = 'git-client' }
+            @{Name = 'git-server' }
+            @{Name = 'github' }
+            @{Name = 'github-api' }
+            @{Name = 'github-branch-source' }
+            @{Name = 'gradle' }
+            @{Name = 'handlebars' }
+            @{Name = 'jackson2-api' }
+            @{Name = 'jdk-tool' }
+            @{Name = 'jjwt-api' }
+            @{Name = 'jquery3-api' }
+            @{Name = 'jsch' }
+            @{Name = 'junit' }
+            @{Name = 'ldap' }
+            @{Name = 'lockable-resources' }
+            @{Name = 'mailer' }
+            @{Name = 'mapdb-api' }
+            @{Name = 'matrix-auth' }
+            @{Name = 'matrix-project' }
+            @{Name = 'momentjs' }
+            @{Name = 'okhttp-api' }
+            @{Name = 'pam-auth' }
+            @{Name = 'pipeline-build-step' }
+            @{Name = 'pipeline-github-lib' }
+            @{Name = 'pipeline-graph-analysis' }
+            @{Name = 'pipeline-input-step' }
+            @{Name = 'pipeline-milestone-step' }
+            @{Name = 'pipeline-model-api' }
+            @{Name = 'pipeline-model-definition' }
+            @{Name = 'pipeline-model-extensions' }
+            @{Name = 'pipeline-rest-api' }
+            @{Name = 'pipeline-stage-step' }
+            @{Name = 'pipeline-stage-tags-metadata' }
+            @{Name = 'pipeline-stage-view' }
+            @{Name = 'plain-credentials' }
+            @{Name = 'plugin-util-api' }
+            @{Name = 'popper-api' }
+            @{Name = 'powershell' }
+            @{Name = 'resource-disposer' }
+            @{Name = 'scm-api' }
+            @{Name = 'script-security' }
+            @{Name = 'snakeyaml-api' }
+            @{Name = 'ssh-credentials' }
+            @{Name = 'ssh-slaves' }
+            @{Name = 'structs' }
+            @{Name = 'subversion' }
+            @{Name = 'timestamper' }
+            @{Name = 'token-macro' }
+            @{Name = 'trilead-api' }
+            @{Name = 'workflow-aggregator' }
+            @{Name = 'workflow-api' }
+            @{Name = 'workflow-basic-steps' }
+            @{Name = 'workflow-cps' }
+            @{Name = 'workflow-cps-global-lib' }
+            @{Name = 'workflow-durable-task-step' }
+            @{Name = 'workflow-job' }
+            @{Name = 'workflow-multibranch' }
+            @{Name = 'workflow-scm-step' }
+            @{Name = 'workflow-step-api' }
+            @{Name = 'workflow-support' }
+            @{Name = 'ws-cleanup' }
+
+        ) {
+            $_.Name -in $plugins | Should -be $true
```

Comment:
You should be able to use `$Name` here... Or....

Instead of passing in an array of Hashtables, you could pass an array of strings, then just us `$_` as the name.